src/blog/features-from-version-5.md

```diff
@@ -0,0 +1,112 @@
+![New Features in Truffle 5.1 and 5.2 Banner](/img/blog/features-from-version-5/blog-header.png)
+
+Truffle 5.1 and 5.2 are exciting releases in the flexibilty they provide to your workflow. Truffle now offers more flexible compilation options by detecting the version of Solidity in your contracts at compile time and by allowing for compilation from ABI files. Because of this, wrangling a project whose contracts use multiple version of Solidity is much easier going forward. We've also enhanced our testing capabilities with Solidity strack traces and better event decoding, making it much easier to get to the source of an error and get richer information from events. When our projects integrate with existing protocols, it's a pain to have to deal with those external contracts. Now Truffle makes this easy with the ability to fetch external contracts verified on Etherscan! Vyper support continues to improve with experimental debugging for Vyper contracts, as well as improved import handling for Vyper.
```

Comment:
Is this first sentence kind of awkward? Would it be better to write something like "...are exciting releases in that there are some new features to help provide more flexibility to your workflow"?